Tests/Parser/Client/fixtures/browser.yml

```diff
@@ -2546,3 +2546,21 @@
     version: "2.0"
     engine: WebKit
     engine_version: 605.1.15
+- 
+  user_agent: Mozilla/5.0 (Linux; Android 8.0.0; SM-G935F Build/R16NW; wv) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/67.0.3396.87 Mobile Safari/537.36 (AirWatch Browser v6.14.0.19)
+  client:
+    type: browser
+    name: VMware AirWatch
+    short_name: VM
+    version: "6.14.0.19"
+    engine: WebKit
+    engine_version: "537.36"
+- 
+  user_agent: AWBrowser/323 CFNetwork/901.1 Darwin/17.6.0
```

Comment:
Where do you know from that `AWBrowser` is VMWare AirWatch and not maybe https://apkpure.com/de/aw-browser-with-chrome-custom-tabs/com.gramvislab.myappslauncher

Reply:
The first page that I visited was [https://www.facebook.com/AWBROWSER/about/](https://www.facebook.com/AWBROWSER/about/) and from there I couldn't see anything related to being a browser. And the other one, the text in this link [https://docs.vmware.com/en/VMware-Workspace-ONE-UEM/1902/Workspace-ONE-UEM-Windows-Phone-Device-Management/GUID-AWT-AWBROWSER-SNIP.html](https://docs.vmware.com/en/VMware-Workspace-ONE-UEM/1902/Workspace-ONE-UEM-Windows-Phone-Device-Management/GUID-AWT-AWBROWSER-SNIP.html) has `AWBROWSER`. I taught it was an older version of it.